src/appdistribution/options-parser-util.ts

```diff
@@ -0,0 +1,56 @@
+import * as fs from "fs-extra";
+import { FirebaseError } from "../error";
+import { needProjectNumber } from "../projectUtils";
+
+/* Takes in comma separated string or a path to a comma/new line separated file
+and converts the input into an string[] of testers or groups. Value takes precedent
+ over file.*/
```

Comment:
JSDoc-type comments for functions have a pretty well defined pattern:

```suggestion
/**
 * Takes in comma separated string or a path to a comma/new line separated file
 * and converts the input into an string[] of testers or groups. Value takes precedent
 * over file.
 */
```

This makes it so that IDEs can show the docs and use the information in it. Definitely apply this to the one below too.